iron-selectable.html

```diff
@@ -245,6 +245,12 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+
+      if (isSelected) {
+        item.setAttribute('aria-selected', 'true');
+      } else {
+        item.removeAttribute('aria-selected');
+      }
```

Comment:
Yeah, TBH I was really unsure which ARIA to set in `menu` vs `selectable`. On the one hand, it seems like the semantics of selection take place in selectable, which suggests that _all_ ARIA related to selection should be set there. On the other hand, selectable seems kind of abstract, without sufficient context to assume the semantic meaning implied by ARIA. So, maybe _all_ ARIA related to selection belongs in menu.

WDYT?